docs/wiki/Ethereum.md

```diff
@@ -0,0 +1,104 @@
+# Ethereum
+
+## Wallet Overview
+
+The Ethereum wallet in the DEX client application is similar to the "native"
+wallets for BTC and DCR, but requires some manual configuration.
+
+Like the other native wallets, the private keys (and accounts) are derived from
+the DEX "application seed", so that reinitializing the DEX app from seed will
+restore the same Ethereum wallet and any ERC20 wallets. The keys are maintained
+exclusively by the application process, and all transaction signing is handled
+by the application.
+
+Unlike the other native wallets, the Ethereum wallet uses "RPC providers"
+specified by the user. This is the wallet's gateway to the Ethereum network.
+Casual ETH users may be familiar with Infura, as an example, since this is the
+default provider for the common MetaMask wallet. When creating your Ethereum
+wallet in the DEX application, you will need to specify the full path to an
+Ethereum RPC provider, i.e. an RPC endpoint.
+
+## Wallet Setup
+
+From the "Wallets" page of the DEX application, you select Ethereum from the
+left menu, which will show you your the ETH balance if you already have a
+wallet, or you will see a "Create a Ethereum Wallet" button:
+
+   <img src="images/eth-create-wallet.png" width="320">
+
+When you click the button, you will be able to specify one or more RPC provider
+endpoints. You can provide a single endpoint, but it is recommended to specify
+more than one for redundancy. Click the "+" icon at the right to add additional
+providers.
+
+   <img src="images/eth-wallet-settings.png" width="320">
+
+The types of endpoints that are supported are:
+
+- (**Recommended**) WebSocket (ws:// or wss:///). These use a persistent
+  connection, which is efficient and supports new block subscriptions to reduce
+  provider utilization and bandwidth.
+- HTTP (http:// or https://). These are less efficient and do not support new
+  block subscriptions, so the client frequently polls for new blocks.
+- IPC. These are typically a special file on your local machine. You must be
+  running a local node to use this, and the IPC file permissions must be set to
+  allow the DEX app to use it. However, typically even a full node will expose a
+  WebSocket interface.
+
+## RPC Provider Suggestions
+
+Consider the following tips when configuring your Ethereum wallet:
+
+- Use **multiple providers**, even if one of them is your own full node. This
+  provides redundancy. Often a provider will fall behind or even stall, so
+  use of multiple providers adds robustness to your wallet.
```

Comment:
I think most users would ignore this recommendation go for single provider (and btw will also be annoyed by that need to configure it on their own unlike with Metamask) without fully realising the risks they are taking. Users might also ignore the recommendation about **private API key** because "it is just a recommendation, and requires additional effort".

Since ignoring these recommendations (providers go down, req limit could be reached) could result in failure to redeem (if all providers fail while user is trying to redeem), how about:
- (bad UX) require at least 2 providers to be configured by user instead of just 1
- (much better UX) allow just 1 provider, but have 2 or more hardcoded fallback providers that will not affect current flow in any way except that these will be used as a measure of last resort when none of user-configured providers are available

Reply:
> But what if I do not what those providers used? What if I'm trying to use one for whatever reason (privacy, security, etc)?

Could potentially have additional opt-in flag "never use default providers" (with the description about this being more privacy-oriented). I think opt-in features are better suited to those people who like to configure their env exactly the way they want, not for average user who might not even want to understand why it is there (assuming DEX at least remotely tries to target such users).

Here is a ticket to continue this discussion:https://github.com/decred/dcrdex/issues/2246.